feature/platform/transceiver/tests/zr_inventory_test/zr_inventory_test.go

```diff
@@ -141,18 +141,27 @@ func opticalChannelComponentFromPort(t *testing.T, dut *ondatra.DUTDevice, p *on
 			t.Fatal("Manual Optical channel name required when deviation missing_port_to_optical_channel_component_mapping applied.")
 		}
 	}
-	compName := gnmi.Get(t, dut, gnmi.OC().Interface(p.Name()).HardwarePort().State())
```

Comment:
instead of fixing this here, I think this should be refactored to use OpticalChannelComponentFromPort from  [github.com/openconfig/featureprofiles/internal/components](http://github.com/openconfig/featureprofiles/internal/components)

@yiwenhu-g WDYT?